oneart

```diff
@@ -0,0 +1,27 @@
+One Art
+Elizabeth Bishop, 1911 - 1979
+
+The art of losing isn’t hard to master;
+so many things seem filled with the intent
+to be lost that their loss is no disaster.
+
+Lose something every day. Accept the fluster
+of lost door keys, the hour badly spent.
+The art of losing isn’t hard to master.
+
+Then practice losing farther, losing faster:
+places, and names, and where it was you meant 
+to travel. None of these will bring disaster.
+
+I lost my mother’s watch. And look! my last, or
+next-to-last, of three loved houses went.
+The art of losing isn’t hard to master.
+
+I lost two cities, lovely ones. And, vaster,
+some realms I owned, two rivers, a continent.
+I miss them, but it wasn’t a disaster.
```

Comment:
oh weird it shows up now. so whether i comment on the code or on the review it shows up in both places or something like that...